addons/website/static/src/components/configurator/configurator.js

```diff
@@ -352,8 +395,12 @@ const actions = {
     changeLogo({state}, data) {
```

Comment:
A bit sad that the original branch was merged without any documentation comments :/ Even just the base of them to act as a separator between functions.

Reply:
Do you want me to take profit of this commit to comment this entire js file?

Reply:
Better to do that in a separate commit, even in another PR if needed